### DISCUSSION TITLE
Re-creating the "paulette" law with Soroban write-up

### DISCUSSION THREAD

**Discussion by heytdep** [stellar/sorobanathon#32] [General]
Hi, @itsfederico and I have written this write-up about writing a fun implementation of the ["Paulette Law"](https://en.wikipedia.org/wiki/Paulette_(tax)) with Soroban along with adding some nice features like putting public offices up for auction by using the [previously-submitted dutch auction contract](https://github.com/stellar/sorobanathon/discussions/27). 

[This is the github repo](https://github.com/Xycloo/soroban-paulette-smart-contract) and the readme contains the write-up to explain what happens in the code and why it happens as well while explaining how some soroban-specific features work.

**Comment by achallagundla** [comment]
You and @itsfederico are on fire!! Looking forward to review your submission and getting back to you soon!

**Comment by achallagundla** [comment]
Heya @heytdep ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission qualifies for a reward of 1500 XLM! 
As always lmk if you have any questions on discord!
